src/components/shared/Button/PaneToggle.js

```diff
@@ -12,17 +12,20 @@ import "./PaneToggle.css";
 type Props = {
   collapsed: boolean,
   handleClick: (string, boolean) => void,
-  horizontal?: boolean,
+  horizontal: boolean,
   position: string
 };
 
 class PaneToggleButton extends Component<Props> {
+  static defaultProps = {
+    horizontal: false
+  };
   shouldComponentUpdate(nextProps: Props) {
-    const { collapsed, horizontal } = this.props;
-
-    return (
-      horizontal !== nextProps.horizontal || collapsed !== nextProps.collapsed
-    );
+    const { collapsed, horizontal, position } = this.props;
+    const diffHorizontal = horizontal !== nextProps.horizontal;
+    const diffCollapsed = collapsed !== nextProps.collapsed;
+    const diffPosition = position !== nextProps.position;
+    return diffHorizontal || diffCollapsed || diffPosition;
   }
```

Comment:
can this be a `PureComponent`?

Reply:
Yeah good point the prop types are simple data types. 👍